content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
-+ [Reference Space Model](/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model/)
-+ [Using Group Selections and Reference Spaces](/docs/user-guide/editor/viewport/group-selections-reference-spaces/)
-+ [Resetting the Transform](/docs/user-guide/editor/viewport/reset-transform/)
-+ [Matching the Transform](/docs/user-guide/editor/viewport/transform-matching/)
-+ [Using Pivot Mode](/docs/user-guide/editor/viewport/pivot-mode/)
-+ [Using the Camera Space](/docs/user-guide/editor/camera-space/)
+![Image of component mode switcher widget](/images/user-guide/editor/component-mode-switcher.png)
+
+Every entity in the 3D viewport has a Transform component.  Other components that you can add to entities have properties that are easily edited in the 3D viewport using the component's _edit mode_.  You can **left-click** the icon of an editable component to enter that component's edit mode.  The Transform component's edit mode is selected by default. 
+
+The following table lists some of the components with an edit mode and describes the tasks you can complete while it is selected in the component mode switcher.  Refer to a component's page in the [Component Reference](/docs/user-guide/components/reference/) for a description of actions you can take while in edit mode.
+
+| Component | Edit Mode Tasks |
+| - | - |
+| [Shape](\docs\user-guide\components\reference\shape) | Edit the dimensions of the various shape components. |
+| [Whitebox](\docs\user-guide\components\reference\shape\white-box) | Design models with the Whitebox tool. |
+| [Spline](\docs\user-guide\components\reference\shape\spline) | Add, remove, and position spline control points. |
+| [Non-uniform Scale](\docs\user-guide\components\reference\non-uniform-scale) | Edit the scaling of an entity on the X, Y, and Z-axis separately. |
+| [PhysX Collider](\docs\user-guide\components\reference\physx\collider) | Edit the dimensions of a collider. |
+| [PhysX Ball Joint](\docs\user-guide\components\reference\physx\ball-joint) | Edit the position and limitations of a joint. |
+
+## Manipulators 
+
+With manipulators, you can edit an entity's Transform component properties directly in the 3D Viewport instead of in the [Entity Inspector](.\..\entity-inspector).  When you select an entity, only the current transform mode's manipulator displays.  The orientation of the manipulator matches the reference space selection.
+
+### Translation manipulator
+
+![Image of the translation manipulator with the linear, planar, and surface manipulator highlighted](/images/user-guide/editor/transform-manipulator.png)
+
+The translation manipulator is composed of three different types of manipulators. The red, green, and blue arrows are _linear manipulators_, they correspond to the X-axis, Y-axis, and Z-axis, respectively.  You can **left-click** and drag a linear manipulator to move an entity along a single axis at a time.  Manipulators turn yellow when you select them.
```

Comment:
uber nit: `Manipulators turn yellow when you select them.` -> `Manipulators turn yellow when you hover over them.`